ras/sosreport.py

```diff
@@ -254,7 +252,7 @@ def test_PPC(self):
         directory_name = tempfile.mkdtemp()
         self.is_fail = 0
         self.run_cmd("sosreport --batch --tmp-dir=%s -o "
-                     "pci,powerpc,procenv,process,processor,kdump" % directory_name)
+                     "pci,powerpc,process,processor,kdump" % directory_name)
```

Comment:
It is good to specify some details of when (which version) and/or why (eg. not supported anymore) the option is being removed, so that it is useful for reference.